src/pages/Folder/components/FolderCategory/index.jsx

```diff
@@ -0,0 +1,49 @@
+import styles from "./styles.module.css";
+import useWindowSize from "hooks/useWindowSize";
+
+export default function FolderCategory({
+  folders,
+  selectedId,
+  onSelectedFolder,
+}) {
+  const [windowSize] = useWindowSize();
+
+  const handleClick = (e) => {
+    onSelectedFolder({ name: e.target.textContent, id: e.target.id });
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.tags}>
+        <span
+          className={`${styles.tag} ${
+            selectedId === null ? styles.selected : ""
+          }`}
+          id={null}
```

Comment:
P3:
id가 없어도 될 것 같아요.

```suggestion
          className={`${styles.tag} ${
            !selectedId ? styles.selected : ""
          }`}
```

Reply:
만약 명시적으로 아이디를 주고 싶으시면 이렇게 하시면 좋을 것 같습니다.

```suggestion
          const SELECTED_ALL_ID = 'selectedAll'
          ...
          className={`${styles.tag} ${
            selectedId === SELECTED_ALL_ID ? styles.selected : ""
          }`}
          id={SELECTED_ALL_ID}
```